todo_app/__init__.py

```diff
@@ -0,0 +1,26 @@
+import os
+from flask import Flask, current_app
+from werkzeug.utils import import_string
+from config import config
+
+
+def create_app(config_name=None, overwrite_board=None):
```

Comment:
This setup is inventive, but we could simplify it and the Config object considerably by taking advantage of dotenv - the tests could then load an appropriate .env file before creating the app, and override e.g. the board name if they need to

This might not be very clear, so do ask if you'd appreciate some help looking at that

Reply:
Yes, I got this from: https://flask.palletsprojects.com/en/1.1.x/config/#configuration-best-practices

As it didn't feel right to have multiple configuration files, but it did end up increasing the complexity.  Will have a go at refactoring it.

Reply:
This one has become a bit of an issue.  For whatever reason, I can't seem to override the board ID, and it always chooses the default one unless I pass the variable into the create_app function.  I have managed to make the integration and unit tests work via dotenv however.  What's the best way to get help with this? Should I upload the latest broken code to review?